README.md

```diff
@@ -1,24 +1,80 @@
-# Rust binding to [Open Quantum Safe][oqs]'s [liboqs][]
+liboqs-rust: Rust bindings for liboqs
+=====================================
 
-This crate provides bindings to [liboqs][]. It provides unsafe `ffi` bindings in the `oqs-sys` crate.
-Safe wrappers are offered via the `oqs` crate.
+[![Build status](https://github.com/dstebila/oqs-rs/workflows/Continuous%20integration/badge.svg)](https://github.com/dstebila/oqs-rs/actions?query=workflow%3A"Continuous+integration")
 
+**liboqs-rust** offers a Rust wrapper for the [Open Quantum Safe](https://openquantumsafe.org/) [liboqs](https://github.com/open-quantum-safe/liboqs/) C library, which is a C library for quantum-resistant cryptographic algorithms.
```

Comment:
This description is slightly inaccurate.  This repository contains *two* Rust crates. There is the `oqs-sys` crate that defines the ``ffi`` interface and there is the ``oqs`` crate that provides the "nice interface" to liboqs, ie. doesn't use C types or `unsafe`

Reply:
See also https://doc.rust-lang.org/cargo/reference/build-scripts.html#-sys-packages

Reply:
I'll clarify to say it offers two Rust wrappers.